include/slam_toolbox/serialization.hpp

```diff
@@ -34,7 +34,7 @@ inline bool fileExists(const std::string & name)
   return stat(name.c_str(), &buffer) == 0;
```

Comment:
Remaining issues are stylistic, you're not matching the rest of the project's style guide. Please make your additions match the surrounding code

Reply:
I've already used ament_cpplint for update code following the surrounding code.